src/main/java/uk/gov/hmcts/darts/annotation/persistence/AnnotationPersistenceService.java

```diff
@@ -39,12 +39,12 @@ public void persistAnnotation(
                 auditApi.recordAudit(IMPORT_ANNOTATION, userIdentity.getUserAccount(), hearingEntity.getCourtCase());
                 var annotation = inboundExternalObjectDirectory.getAnnotationDocumentEntity().getAnnotation();
                 hearing.get().addAnnotation(annotation);
+                hearingRepository.save(hearingEntity);
             }
-
         );
 
-        externalObjectDirectoryRepository.save(inboundExternalObjectDirectory);
-        externalObjectDirectoryRepository.save(unstructuredExternalObjectDirectory);
+        externalObjectDirectoryRepository.saveAndFlush(inboundExternalObjectDirectory);
+        externalObjectDirectoryRepository.saveAndFlush(unstructuredExternalObjectDirectory);
```

Comment:
SaveAndFlush isn't required here, the method is transactional and these are the last statements.  The changes are flushed by the commit.

Reply:
ok